src/components/layouts/featurelist/feature-list-content.tsx

```diff
@@ -0,0 +1,450 @@
+          title: 'Integrations',
+          description: (
+            <>            
+            Payments integration powered by Stripe as well as Email integration to send out emails in bulk or individually.
+            </>
+          ),
+          icon: PlusCircle
+        }
+      ]
+    }
+  ]
+}
+
+export const featureListKratos = {
+features : [
+  {
+    icon: CircleThreePlus,
+    title: 'Ory Kratos Features',
+    description: (
+      <>
+      Most software applications need to deal with user and identity management. Ory Kratos serves common flows so you dont need to rewrite everything yourself.
+      </>
+    ),
+    button: (
+      <Button
+      style={'link'} to={'/docs/kratos/quickstart/'} iconRight={ArrowRight} > Get started </Button>
+    ),
+    features: [
+      {
+        title: 'Self Service Login and Registration',
+        description: (
+          <>
+          Enable users to create and sign in to accounts using username/email and password combinations, Social Login, passwordless flows and more.
+          </>
+        ),
+        icon: CheckCircle
+        },
+        {
+          title: 'User Management',
+          description: (
+          <>
+          Manage your userbase and get, create, update or delete identities and their data.
+          </>
+        ),
+        icon: CheckCircle
+        },
+        {
+          title: 'Bring Your Own UI and Identity Models',
+          description: (
+          <>
+          Use customizable identity models (add fields like name, address, favourite pet ect..) and create your own user interfaces for flows in your style and branding.
+          </>
+        ),
+        icon: CheckCircle
+        },
+        {
+          title: 'Profile and Account Management',
+          description: (
+          <>
+          Let your users use secure flows to update passwords, personal details, email addresses, linked social profiles and more.
+          </>
+        ),
+          icon: CheckCircle
+        },
+        {
+          title: 'Social Logins',
+          description: (
+          <>
+          Simplify UX and let them use their existing accounts at Google, GitHub, etc. to sign up and log in. All OIDC providers are supported.
+          </>
+        ),
+          icon: CheckCircle
+        },
+        {
+          title: 'Account Verification and Recovery',
+          description: (
+          <>
+          Verify an identity by checking email, phone number, or physical address of that user. Provide recovery of accounts using "Forgot Password" flows, security codes, ect.
+          </>
+        ),
+          icon: CheckCircle
+        }
+      ]
+    }
+  ]
+}
+
+export const featureListHydra  = {
+  features : [
+    {
+      icon: CircleThreePlus,
+      title: 'Ory Hydra Features',
+      description: (
+        <>
+        Issue OAuth 2.0 Access, Refresh and ID tokens in a secure and efficient manner. Ory Hydra secures critical environments for billions of users.
```

Comment:
```suggestion
        Issue OAuth 2.0 Access, Refresh and ID Tokens in a secure and efficient manner. Ory Hydra secures critical environments for billions of users.
```

Reply:
There are two "secure" following one another, please avoid duplication. Generally, by-sentences like "efficient manner" add little information. Maybe it makes sense to rework this sentence:

```suggestion
        Ory Hydra is a hardened and certified OAuth 2.0 and OpenID Connect provider, securing hundreds of billions of API requests in thousands of deployments.
```